matchbox_socket/src/webrtc_socket/wasm.rs

```diff
@@ -516,14 +522,14 @@ fn create_data_channel(
     leaking_channel_event_handler(
         |f| channel.set_onerror(f),
         move |event: Event| {
-            error!("Error in data channel: {:?}", event);
+            error!("Error in data channel: {event:?}");
```

Comment:
one nitpick I have is that most logged message start with a lowercase character, while this one starts with an uppercase. I would like us to be consistent?